packages/cloudflare/package.json

```diff
@@ -59,6 +59,7 @@
     "vitest": "catalog:"
   },
   "dependencies": {
+    "@opennextjs/aws": "https://pkg.pr.new/@opennextjs/aws@3b372b6",
```

Comment:
I'll make a release tomorrow morning, you'll be able to use 3.2.1 instead of this

Reply:
Thanks! 
As explained above, a npm release isn't a req for us before we merge back to main.